### PR TITLE
bugfix/pdpdevtool-4376: Import Files / Update File bugfixes and improvements

### PR DESCRIPTION
**Pull request by davidecorreu**
It has the latest improvements for: PDPDEVTOOL-4023: Import Files in vscode.
It solves:
PDPDEVTOOL-4376
PDPDEVTOOL-4381
PDPDEVTOOL-4382
PDPDEVTOOL-4383
PDPDEVTOOL-4384

Commit summary:
add revised messages for ImportFiles.ts
add generic warning message in MessageService.ts
show warning message if any imported file fails
fix excludeproperties parameter in ImportFiles.ts
add new UpdateFile command
extract common logic to FileImportCommon.ts
rename update single object command

### CODE REVIEW

File: packages/vscode-extension/messages.json [hunk 1]
```diff
@@ -66,15 +68,15 @@
 	"EXTENSION_PROJECT_STARTUP_MESSAGE_PROJECT_NEEDS_SETUP_ACCOUNT": "No account has been set up for this project. Click \"SuiteCloud: Setup Account\" to link your project with your account.",
 
 	"IMPORT_FILES_ERROR_IMPORT_TO_SUITEAPP_NOT_ALLOWED": "Files cannot be imported to a SuiteApp project. You can only import files to Account Customization Projects.",
-	"IMPORT_FILES_FINISHED": "The importing process has finished.",
-	"IMPORT_FILES_IMPORTING_FILE": "Importing file...",
+	"IMPORT_FILES_ERROR_NOT_ALLOWED_FOLDER": " Files can only be imported from \"/FileCabinet/SuiteScripts\", \"/FileCabinet/Templates/E-mail Templates\", \"/FileCabinet/Templates/Marketing Templates\" and \"/FileCabinet/Web Site Hosting Files\" folders in an Account Customization Project.",
```
Comment (antoniofruci) [inline packages/vscode-extension/messages.json:71]: imho it sounds better if we rephrase it like: "In an Account Customization Project files can only be imported from..."

**Review by antoniofruci — APPROVED** [review]

**Review by karl-anthony-ng — APPROVED** [review]

File: packages/vscode-extension/messages.json (hunk 1 above)
Comment (davidecorreu) [reply packages/vscode-extension/messages.json:71]: This message still needs to be reviewed by @Montsant19 . We'll take into consideration your comment.

Comment (Montsant19) [reply packages/vscode-extension/messages.json:71]: I don't think using this adverbial of place at the beginning sounds natural in English. We will review it again and let you know, @antoniofruci. Thanks!

Comment (davidecorreu) [reply packages/vscode-extension/messages.json]: New message:
**Files in an Account Customization Project can only be imported from the following File Cabinet folders: "/SuiteScripts", "/Templates/E-mail Templates", "/Templates/Marketing Templates" and "/Web Site Hosting Files".**

**Pull request merged**